### ISSUE TITLE
"polars" is required in new version

### ISSUE THREAD

**Issue by PavelAgurov** [Sinaptik-AI/pandas-ai#982]
### System Info

pandas 2.0.2, windows

### 🐛 Describe the bug

I have error: 

```
ModuleNotFoundError: No module named 'polars'
Traceback:
File "\.env\Lib\site-packages\streamlit\runtime\scriptrunner\script_runner.py", line 535, in _run_script
    exec(code, module.__dict__)
File "\main.py", line 13, in <module>
    from pandasai.llm.openai import OpenAI
File "\.env\Lib\site-packages\pandasai\__init__.py", line 7, in <module>
    from pandasai.smart_dataframe import SmartDataframe
File "\.env\Lib\site-packages\pandasai\smart_dataframe\__init__.py", line 26, in <module>
    from pandasai.agent.base import Agent
File "\.env\Lib\site-packages\pandasai\agent\__init__.py", line 1, in <module>
    from .base import Agent
File "\.env\Lib\site-packages\pandasai\agent\base.py", line 8, in <module>
    from pandasai.pipelines.chat.chat_pipeline_input import (
File "\.env\Lib\site-packages\pandasai\pipelines\__init__.py", line 3, in <module>
    from .pipeline import Pipeline
File "\.env\Lib\site-packages\pandasai\pipelines\pipeline.py", line 8, in <module>
    from pandasai.helpers.query_exec_tracker import QueryExecTracker
File "\.env\Lib\site-packages\pandasai\helpers\query_exec_tracker.py", line 10, in <module>
    from pandasai.connectors import BaseConnector
File "\.env\Lib\site-packages\pandasai\connectors\__init__.py", line 10, in <module>
    from .polars import PolarsConnector
File "\.env\Lib\site-packages\pandasai\connectors\polars.py", line 9, in <module>
    import polars as pl
```

**Comment by PavelAgurov** [comment]
Not sure how to remove pl dependency in the best way. Found df_info.py - there is logic here already. Any ideas?

**Comment by akashAD98** [comment]
facing the same issue for installing it

**Comment by margauxesq** [comment]
python 3.11.8
pandas 2.0.3
→ ModuleNotFoundError: No module named 'polars'

**Comment by philchass** [comment]
python 3.11.8
pandas 2.0.3
→ ModuleNotFoundError: No module named 'polars'

**Comment by PavelAgurov** [comment]
As I see bug has been fixed in 2.0.16.
